test/FastTests/Client/Indexing/TimeSeries/BasicTimeSeriesIndexes.cs

```diff
@@ -55,9 +55,9 @@ public void BasicMapIndex(Options options)
                     "   User = ts.DocumentId " +
                     "}" }
                 }));
-
+            //   WaitForUserToContinueTheTest(store);
                 var staleness = store.Maintenance.Send(new GetIndexStalenessOperation("MyTsIndex"));
-                Assert.True(staleness.IsStale);
+               // Assert.True(staleness.IsStale);
```

Comment:
Commented assertion? Here and below as well

Reply:
Ouch, I temporarily commented it out and forgot to uncomment it. I have reverted that. 👍